16x9/DialogFullScreenInfo.xml

```diff
@@ -19,6 +20,9 @@
 			<ondown condition="!VideoPlayer.Content(livetv) + Skin.HasSetting(Enable.UsePausedOverlay)">Skin.ToggleSetting(ShowSeekbar)</ondown>
 			<onclick condition="VideoPlayer.Content(livetv)">Skin.ToggleSetting(TogglePlotLiveTV)</onclick>
 			<onup condition="VideoPlayer.Content(livetv)">Skin.ToggleSetting(Enable.LiveTVFullPlot)</onup>
+			<onclick condition="VideoPlayer.Content(livetv) + !Skin.HasSetting(ToggleLiveTVNext)">Skin.ToggleSetting(ToggleLiveTVNext)</onclick>
```

Comment:
No, it is working for me (with WAF)
I have this code in the skin since about 2 months without creating a PR ;)